src/main/kotlin/com/hedvig/rapio/externalservices/memberService/dto/IsMemberRequest.kt

```diff
@@ -0,0 +1,7 @@
+package com.hedvig.rapio.externalservices.memberService.dto
+
+data class IsMemberRequest(
+    val memberId: String?,
+    val ssn: String?,
+    val email: String?
```

Comment:
If we use `LogCall`
```suggestion
    @Masked val ssn: String?,
    @Masked val email: String?
```

As a reminder for myself. Let's not forget to add these on `InsuranceAddress` and some other DTO I added before 🙈